support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -48,12 +79,12 @@ export function CheckoutNudgeContainer({
 	return renderNudge({
 		contributionType,
 		nudgeDisplay: displayNudge,
-		nudgeTitleCopySection1: 'Make a bigger impact',
-		nudgeTitleCopySection2: `Support us every ${
-			recurringType === 'MONTHLY' ? 'month' : 'year'
-		}`,
-		nudgeParagraphCopy: paragraphCopyVariant,
+		nudgeTitleCopySection1: titleNudge,
+		nudgeTitleCopySection2: title2Nudge,
+		nudgeParagraphCopy: paragraphNudge,
```

Comment:
Just a suggestion but could also simplify these prop names 

```suggestion
		nudgeTitle: ...,
		nudgeSubtitle: ...,
		nudgeParagraphCopy: ...,
```

Reply:
Ah yes, subtitle was the name I was looking for, thanks, changed to match